courses/free-courses-en.md

```diff
@@ -119,6 +120,11 @@
 * [Programming Mobile Services for Android Handheld Systems: Concurrency](https://www.coursera.org/course/posaconcurrency)
 
 
+### Artificial Intelligence
+
+* [Elements of AI](https://www.elementsofai.com) - University of Helsinki, Reaktor
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category.